lib/Amazon/MiqEc2InstanceTypes.rb

```diff
@@ -987,6 +987,30 @@ module MiqEc2InstanceTypes
 
   # Types that are no longer advertised
   DISCONTINUED_TYPES = {
+    "unknown" => {
+      :disabled                => true,
+      :name                    => "unknown",
+      :family                  => "unknown",
+      :description             => "unknown",
+      :memory                  => 0.megabytes,
+      :vcpu                    => 1,
+      :ebs_only                => true,
+      :instance_store_size     => 0,
+      :instance_store_volumes  => 0,
+      :architecture            => [:x86_64],
+      :virtualization_type     => [:hvm],
```

Comment:
I'm wondering if this should be empty.  That way, it can't be used in provisioning.